src/Uno.UI.Runtime.Skia.Wpf/Input/HostPointerHandler.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Is this needed?

Reply:
If you mean the class then yes, but no for the header 👍